templates/layout_logged_in.html

```diff
@@ -1,22 +1,10 @@
 <!doctype html>
-<title>PaintingTrivia</title>
-<div class="page">
-    <h2>Hello there, {{ username }}</h2>
-
-    <form method="post">
-        <p>
-            <button type="submit" name="navigate_to" value="logout">Logout</button>
-        <p>
-            <button type="submit" name="navigate_to" value="main_menu">Back to menu</button>
-    </form>
-</div>
-<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='style.css') }}">
-<script src="{{ url_for('static', filename='js/libs/jquery-2.2.0.min.js') }}"></script>
-<script src="{{ url_for('static', filename='js/app/app.js') }}"></script>
-<div class="page">
-    <h1>PaintingTrivia</h1>
-    {% for message in get_flashed_messages() %}
-        <div class="flash">{{ message }}</div>
-    {% endfor %}
-    {% block body %}{% endblock %}
-</div>
\ No newline at end of file
+<html>
```

Comment:
could you please merge layout and layout_logged_in into one file (you need to check in the logged in version if some flag in session is set)